src/TLuaInterpreter.cpp

```diff
@@ -10217,86 +10217,151 @@ int TLuaInterpreter::getCmdLine(lua_State* L)
 
 int TLuaInterpreter::installPackage(lua_State* L)
 {
-    string event;
+    QString packageName;
     if (!lua_isstring(L, 1)) {
-        lua_pushstring(L, "installPackage(): wrong argument type");
-        lua_error(L);
-        return 1;
+        lua_pushfstring(L, "installPackage: bad argument #1 type (pathFileName of package as string expected, got %s!)",
+                        luaL_typename(L, 1));
+        return lua_error(L);
     } else {
-        event = lua_tostring(L, 1);
+        packageName = QString::fromUtf8(lua_tostring(L, 1));
+        if (packageName.isEmpty()) {
+            lua_pushnil(L);
+            lua_pushstring(L, "An empty string is not a valid pathFileName for a package to install.");
```

Comment:
`not a valid path`

Reply:
A __path__ is not a __file__ name to my mind - I used this term to clearly indicates that both elements are required here (compared to some of the related lua functions that only takes a package _name_). I note the Qt `QDir` and `QFile` use similar forms in some of their methods IMHO.

Reply:
It might be clear to you as a hardcore programmer but it won't be to most users who are just hacking with this... the version I gave will be more understandable.

Reply:
As above, I've decided to compromised on `An empty string is not a valid path and file name for a package to install.`